README.md

```diff
@@ -109,8 +109,11 @@ The templates use some Host Parameters to control the flow of the template. Thes
 * `partitioning-crypto-password`: Password for luks crypto method. Recommend changing this post install! (default: `temporarypassword`)
 * `partitioning-crypto-password-weak`: Allow weak passwords when using crypto. (default: `false`)
 * `partitioning-allow-noswap`: Allow partitioning without swap. (default: `false`)
-* `enable-puppetlabs-repo`: Add the Puppet Labs APT repo to the APT sources during install (default: `false`)
-* `enable-puppetlabs-pc1-repo`: Add the Puppet Labs PC1 APT repo to the APT sources during install (default: `false`)
+* `enable-puppet-puppet5`: Assume Puppet 5 AIO packages instead of system packages
+* `enable-puppet-puppet6`: Assume Puppet 6 AIO packages instead of system packages
+* `enable-puppetlabs-repo`: Add the Puppet Labs repo during install (default: `false`)
```

Comment:
This was the 3.x PL repos, iirc, so this should be removable now...

Reply:
I was still wondering about it since it does appear to exist. It appears it's now puppet 6 so I think it'll be the latest supported version.

Reply:
Interesting, didn't know that... Then it's fine :+1: